data/monster/Bio-Elementals/squidgy_slime.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
encoding="ISO-8859-1"

Reply:
It's not that important but we should stick to one standard

Reply:
yes I talked about it with @DSpeichert but we need to get to a standard one first